app/src/main/java/com/kickstarter/extensions/SnackbarExt.kt

```diff
@@ -0,0 +1,16 @@
+package com.kickstarter.extensions
+
+import android.content.Context
+import android.support.design.widget.Snackbar
+import android.support.v4.content.ContextCompat
+import android.support.v4.view.ViewCompat
+import android.view.ViewGroup
+import com.kickstarter.R
+
+fun Snackbar.success(context: Context) {
+    val params = this.view.layoutParams as ViewGroup.MarginLayoutParams
```

Comment:
Magic numbers make code harder to understand.
Let's try something like this instead:
```   
    val grid1 = context.resources.getDimensionPixelSize(R.dimen.grid_1)
    val grid2 = context.resources.getDimensionPixelSize(R.dimen.grid_2)
    params.setMargins(grid1, 0, grid1, grid2)
```